src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -446,9 +446,9 @@ define(['app-config', 'config', 'sulucontent/components/content/preview/main'],
         },
 
         submit: function(action) {
-            this.sandbox.emit('sulu.header.toolbar.item.loading', 'save');
             this.dfdListenForResourceLocator.then(function() {
                 if (this.sandbox.form.validate(this.formId)) {
+                    this.sandbox.emit('sulu.header.toolbar.item.loading', 'save');
```

Comment:
Putting this line into the resolved promise fixes the issue https://github.com/sulu-io/sulu-standard/issues/603#event-550487055 but causes #2007. Didn't want to change this code too much, since I don't want to introduce new problems.